Himanshu_Kumar/UsersTask/DbModels/csharpTaskContext.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace UsersTask.DbModels
+{
+    public partial class csharpTaskContext : DbContext
+    {
+        public csharpTaskContext()
+        {
+        }
+
+        public csharpTaskContext(DbContextOptions<csharpTaskContext> options)
+            : base(options)
+        {
+        }
+
+        public virtual DbSet<Userdetails> Userdetails { get; set; }
+        public IEnumerable<object> UserDetails { get; internal set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
```

Comment:
Please Explain  why is Connection string mentioned here when we are Using Dependency Injection for the Context.